### ISSUE TITLE
Review_AndroidTodo

### ISSUE THREAD

**Issue by hirokit** [hirokit/Todo#1]
My app is complete, please review /cc @codepathreview and @codepath

**Comment by nesquena** [comment]
Hi Hiroki,

Thanks for applying and submitting your pre-work. Can you please review the section in the prework about [the readme](http://courses.codepath.com/snippets/intro_to_android/prework#heading-4-submitting-the-project) for your project? In particular, check out this [README template](http://courses.codepath.com/snippets/intro_to_android/readme_templates/prework_readme.md?raw=true) and be sure to [include a GIF recorded with LiceCap](http://www.cockos.com/licecap/). Please ping me back here once that's complete.

Based on your LinkedIn it is unclear if you meet the [selection criteria for participating in the course](http://courses.codepath.com/snippets/intro_to_android/selection_criteria). In particular:
- Within past 3 years have been employed as a software engineer, team lead, technical founder, VPE
- 2+ years of professional software development experience (average participant has 5+ years)
- Sustained professional software experience at companies involving shipping production code strongly preferred

Can you review that and let me know if you can confirm that you meet the requirements? If not, I can add you as a "remote self-paced observer" so you can still have access to the curriculum and learn at your pace. In the meantime, you should also check out the [resources we provide here](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me).

**Comment by hirokit** [comment]
Hi Nathan,

Thanks for your reply. As for my experience, I was managing projects to develop internal trading tools in the bank.
I was analyst so technically software engineering was not my job title. 
Anyway I did that for about 2 years. Plus I worked on Java based projects which involve developing social media platform and mobile games while I was at Stanford.
After I switched to social media industry, I kept working on developing web apps (on RoR) including founding my own project. (2+ years)
So I believe I have what it takes to participate. 
However, if you think this is not enough, I am happy to be added on the observer assuming I can follow the project everyone is working on remotely.

In that case, I was wondering if I could sit even only for the first day or just come say hi to people at FB MPK since I will be there anyway?

**Comment by nesquena** [comment]
Ok thanks for the explanation. I would be happy to add you as an observer. I have marked you down as a remote observer so you can learn self-paced with the course materials and project once the course begins alongside other observers. More details will follow closer to the class. In the meantime you may also enjoy reviewing our [comprehensive open-source Android Guides](http://guides.codepath.com/android#getting-started) for lots of resources and links to other Android courses and materials as well.

> In that case, I was wondering if I could sit even only for the first day or just come say hi to people at FB MPK since I will be there anyway?

Yes, since you worked at Facebook, you are welcome to come by to say "Hi" for the first session if you are already there :) But not for subsequent sessions, although you'll have access to the content online as an observer.

**Comment by hirokit** [comment]
Thanks for adding me as an observer. Yes I will just stop by the very first class to introduce myself. 
Quick questions on remote,
1. Will I be receiving some kind of login&PW in order to access these materials and projects people are working on?
2. Will I be able to collaborate with some other people at some point?

**Comment by nesquena** [comment]
> 1. Will I be receiving some kind of login&PW in order to access these materials and projects people are working on?

Yes, you will be able to access all the content through our learning portal. More details soon.

> 1. Will I be able to collaborate with some other people at some point?

Yep! There are 20+ observers that will all be working through the material together, can ask each other questions and even group into remote teams.

**Comment by hirokit** [comment]
Cool thanks. Will wait for receiving the access to the portal then!
